banyan/resource_service_db.go

```diff
@@ -211,6 +211,12 @@ func DbSchema() map[string]*schema.Schema {
 			Default:     true,
 			Description: "Allow the end user to override the backend_port for this service",
 		},
+		"policy_enforcing": {
```

Comment:
@rohitSangamnerkar  require to add testcase to cover this scenario, Testcases should cover:
1) Setting true/false
2) Import 
3) Pre existing template without policy_enforcing set, (where it is presumed to be true) and import the same to make sure config is not lost.